src/components/Users/UserListAndCard.tsx

```diff
@@ -111,37 +97,45 @@ export const UserStatusIndicator = ({
 }) => {
   const authUser = useAuthUser();
   const isAuthUser = user.id === authUser.external_id;
-  const isOnline = isUserOnline(user) || isAuthUser;
   const { t } = useTranslation();
 
   return (
-    <div
-      className={classNames(
-        "inline-flex items-center gap-2 rounded-full",
-        addPadding ? "px-3 py-1" : "py-px",
-        isOnline ? "bg-green-100" : "bg-gray-100",
-        className,
-      )}
+    <span
+      title={`${new Date(user.last_login).toLocaleString()}`}
+      className={addPadding ? "px-3 py-1" : "py-px"}
     >
-      <span
-        className={classNames(
-          "inline-block h-2 w-2 shrink-0 rounded-full",
-          isOnline ? "bg-green-500" : "bg-gray-400",
-        )}
-      ></span>
-      <span
-        className={classNames(
-          "whitespace-nowrap text-xs",
-          isOnline ? "text-green-700" : "text-gray-500",
-        )}
-      >
-        {isOnline
-          ? t("online")
-          : user.last_login
-            ? relativeTime(user.last_login)
-            : t("never")}
-      </span>
-    </div>
+      {isUserOnline(user) || isAuthUser ? (
+        <Badge variant="secondary" className="bg-green-100 whitespace-nowrap">
+          <span className="inline-block h-2 w-2 shrink-0 rounded-full bg-green-500 mr-2" />
+          <span className="text-xs text-green-700">{t("online")}</span>
+        </Badge>
+      ) : user.last_login ? (
+        <Badge variant="secondary" className="bg-yellow-100 whitespace-nowrap">
+          <span className="inline-block h-2 w-2 shrink-0 rounded-full bg-yellow-500 mr-2" />
+          <span className="text-xs text-yellow-700">
+            {t("last_seen")}{" "}
+            {(() => {
+              const diffInMinutes = dayjs().diff(user.last_login, "minutes");
+              const diffInHours = dayjs().diff(user.last_login, "hours");
+              const diffInDays = dayjs().diff(user.last_login, "days");
+
+              if (diffInMinutes < 60) {
+                return `${diffInMinutes} ${t("minutes_ago")}`;
+              } else if (diffInHours < 24) {
+                return `${diffInHours} ${t("hour")}${diffInHours > 1 ? "s" : ""} ${t("ago")}`;
+              } else {
+                return `${diffInDays} ${t("days") + t("ago")}`;
+              }
```

Comment:
we already have a util called `relativeTime` that does this. re-use it instead of reinventing it